Sources/Plasma/FeatureLib/pfDXPipeline/plDXPipeline.cpp

```diff
@@ -753,7 +753,7 @@ void    plDXPipeline::IInitDeviceState()
     fD3DDevice->SetRenderState( D3DRS_MULTISAMPLEANTIALIAS, ( fSettings.fD3DCaps & kCapsFSAntiAlias ) ? TRUE : FALSE );
     fD3DDevice->SetRenderState( D3DRS_ANTIALIASEDLINEENABLE,        FALSE );
 
-    fD3DDevice->SetRenderState( D3DRS_DITHERENABLE,     FALSE );
+    fD3DDevice->SetRenderState( D3DRS_DITHERENABLE,     TRUE );
```

Comment:
Not sure if this is supposed to be here - but this bool has been flipped which changes the rendering features.

Reply:
oh thanks for catching that i had no clue that got lost in my edits